### PR TITLE
skip gluster transport protocol 'rdma' case

### PR DESCRIPTION
**Pull request by chunfuwen**
Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/virtual_disks/virtual_disks_gluster.py [hunk 1]
```diff
@@ -12,6 +12,8 @@
 from virttest.libvirt_xml import vm_xml
 from virttest.libvirt_xml.devices.disk import Disk
 
+from provider import libvirt_version
```
Comment (Yingshun) [inline libvirt/tests/src/virtual_disks/virtual_disks_gluster.py]: suggest to use libvirt_version in avocado-vt/virttest.

**Comment by chunfuwen** [comment]
paste log
'''
2020-03-06 15:09:33,195 process          L0714 INFO | Command 'virsh list' finished with 0 after 0.5056145191192627s
2020-03-06 15:09:33,219 test             L0270 INFO | cleaning libvirtd logs...
2020-03-06 15:09:33,221 stacktrace       L0041 ERROR| 
2020-03-06 15:09:33,221 stacktrace       L0044 ERROR| Reproduced traceback from: /var/tmp/libvirt-ci/runtest/avocado-vt/avocado/avocado/core/test.py:844
2020-03-06 15:09:33,221 stacktrace       L0047 ERROR| Traceback (most recent call last):
2020-03-06 15:09:33,221 stacktrace       L0047 ERROR|   File "/var/tmp/libvirt-ci/runtest/avocado-vt/avocado-vt/avocado_vt/test.py", line 301, in runTest
2020-03-06 15:09:33,221 stacktrace       L0047 ERROR|     raise self.__status  # pylint: disable=E0702
2020-03-06 15:09:33,221 stacktrace       L0047 ERROR|   File "/var/tmp/libvirt-ci/runtest/avocado-vt/avocado-vt/avocado_vt/test.py", line 252, in setUp
2020-03-06 15:09:33,221 stacktrace       L0047 ERROR|     self._runTest()
2020-03-06 15:09:33,222 stacktrace       L0047 ERROR|   File "/var/tmp/libvirt-ci/runtest/avocado-vt/avocado-vt/avocado_vt/test.py", line 425, in _runTest
2020-03-06 15:09:33,222 stacktrace       L0047 ERROR|     run_func(self, params, env)
2020-03-06 15:09:33,222 stacktrace       L0047 ERROR|   File "/var/lib/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-libvirt/libvirt/tests/src/virtual_disks/virtual_disks_gluster.py", line 214, in run
2020-03-06 15:09:33,222 stacktrace       L0047 ERROR|     libvirt.check_exit_status(cmd_result)
2020-03-06 15:09:33,222 stacktrace       L0047 ERROR|   File "/var/tmp/libvirt-ci/runtest/avocado-vt/avocado-vt/virttest/utils_test/libvirt.py", line 1588, in check_exit_status
2020-03-06 15:09:33,222 stacktrace       L0047 ERROR|     raise exceptions.TestFail(results_stderr_52lts(result))
2020-03-06 15:09:33,222 stacktrace       L0047 ERROR| avocado.core.exceptions.TestFail: error: Failed to attach device from /var/tmp/xml_utils_temp_4wix0f08.xml
2020-03-06 15:09:33,222 stacktrace       L0047 ERROR| error: internal error: transport protocol 'rdma' is not yet supported
2020-03-06 15:09:33,222 stacktrace       L0047 ERROR| 
2020-03-06 15:09:33,222 stacktrace       L0048 ERROR| 
2020-03-06 15:09:33,222 test             L0849 DEBUG| Local variables:
2020-03-06 15:09:33,253 test             L0852 DEBUG|  -> self <class 'avocado_vt.test.VirtTest'>: 1-type_specific.io-github-autotest-libvirt.virtual_disks.gluster.hot_plug.disk_qcow2.attach-device.transport_rdma
'''

File: libvirt/tests/src/virtual_disks/virtual_disks_gluster.py (hunk 1 above)
Comment (chunfuwen) [reply libvirt/tests/src/virtual_disks/virtual_disks_gluster.py]: updated

**Comment by chunfuwen** [comment]
@hahakiki2010 ,please help confirm the change

**Comment by chunfuwen** [comment]
paste test result here
'''
Reproduced traceback from: /root/code/avocado/avocado/core/test.py:841
Traceback (most recent call last):
  File "/root/code/avocado-vt/avocado_vt/test.py", line 301, in runTest
    raise self.__status  # pylint: disable=E0702
  File "/root/code/avocado-vt/avocado_vt/test.py", line 252, in setUp
    self._runTest()
  File "/root/code/avocado-vt/avocado_vt/test.py", line 425, in _runTest
    run_func(self, params, env)
  File "/root/code/tp-libvirt/libvirt/tests/src/virtual_disks/virtual_disks_gluster.py", line 169, in run
    test.cancel("transport protocol 'rdma' is not yet supported")
  File "/root/code/avocado/avocado/core/test.py", line 1057, in cancel
    raise exceptions.TestCancel(message)
avocado.core.exceptions.TestCancel: transport protocol 'rdma' is not yet supported

CANCEL 1-type_specific.io-github-autotest-libvirt.virtual_disks.gluster.hot_plug.disk_qcow2.attach-device.transport_rdma -> TestCancel: transport protocol 'rdma' is not yet supported

Requested MAC address release from persistent vm avocado-vt-vm1. Ignoring.
Test results available in /root/avocado/job-results/job-2020-03-12T08.38-f5e5521
found extension EntryPoint.parse('json = avocado.plugins.jsonresult:JSONResult')
found extension EntryPoint.parse('xunit = avocado.plugins.xunit:XUnitResult')

'''

**Pull request merged**